src/Facebook/FacebookBatchRequest.php

```diff
@@ -62,20 +62,21 @@ public function __construct(FacebookApp $app = null, array $requests = [], $acce
     }
 
     /**
-     * A a new request to the array.
+     * Adds a new request to the array.
      *
      * @param FacebookRequest|array $request
      * @param string|null           $name
+     * @param string[]              $options The batch options e.g. 'omit_response_on_success'
      *
      * @return FacebookBatchRequest
      *
      * @throws \InvalidArgumentException
      */
-    public function add($request, $name = null)
+    public function add($request, $name = null, array $options = [])
```

Comment:
Yeah, I'm curious if we can even add this to 5.4 without adding a BC break. Can you think of a way outside of adding a new method for "adding with options" or something?